itm_minimal_runner.py

```diff
@@ -48,22 +49,10 @@
 from swagger_client.configuration import Configuration
 from swagger_client.api_client import ApiClient
 from swagger_client.models import Scenario, State, AlignmentTarget, Action, Casualty
-
-class ActionType(Enum):
-    APPLY_TREATMENT = "APPLY_TREATMENT"
-    CHECK_ALL_VITALS = "CHECK_ALL_VITALS"
-    CHECK_PULSE = "CHECK_PULSE"
-    CHECK_RESPIRATION = "CHECK_RESPIRATION"
-    DIRECT_MOBILE_CASUALTIES = "DIRECT_MOBILE_CASUALTIES"
-    END_SCENARIO = "END_SCENARIO"
-    MOVE_TO_EVAC = "MOVE_TO_EVAC"
-    SITREP = "SITREP"
-    TAG_CASUALTY = "TAG_CASUALTY"
-
-    def __new__(cls, type):
-        obj = object.__new__(cls)
-        obj._value_ = type
-        return obj
+from swagger_client.models.action_type import ActionType
```

Comment:
The `Enum` import isn't used.  Looks like that was there from before this ticket.